src/lib/Preview.js

```diff
@@ -1882,6 +1880,21 @@ class Preview extends EventEmitter {
         const fileId = typeof fileIdOrFile === 'string' ? fileIdOrFile : fileIdOrFile.id;
         return getProp(this.previewOptions, `fileOptions.${fileId}.${optionName}`);
     }
+
+    /**
+     * Refresh the access token
+     *
+     * @private
+     * @return {Promise<string|Error>}
+     */
+    refreshToken = () => {
+        if (typeof this.previewOptions.token !== 'function') {
+            return Promise.reject(new Error('Token is not a function and cannot be refreshed.'));
```

Comment:
Was the token originally a string and you're converting it to a function that has to be passed in?

Reply:
User can pass a token or a string. In BUIE, a string was passed in Preview originally and I converted the string to a generation function in this ticket: https://github.com/box/box-ui-elements/pull/1689.